tools/autograd/derivatives.yaml

```diff
@@ -995,11 +995,11 @@
   self: std_backward(result, grad, self, dim, unbiased, keepdim)
 
 - name: sub.Tensor(Tensor self, Tensor other, *, Scalar alpha=1) -> Tensor
-  self: grad
-  other: -grad * alpha
+  self: handle_r_to_c(self.scalar_type(), grad)
```

Comment:
why do we need this?  This seems pretty similar to https://github.com/pytorch/pytorch/blob/e02a3e190ec79b2ac9a4fbc3428473503889c690/torch/csrc/autograd/engine.cpp#L611 -- is it because `to` doesn't work c->r?

Reply:
.to works for `C->R`. currently `handle_r_to_c` returns a (real) view of the final gradient value in case the gradient is complex and the input is not.